sdks/java/extensions/sql/src/test/java/org/apache/beam/sdk/extensions/sql/integrationtest/BeamSqlComparisonOperatorsIntegrationTest.java

```diff
@@ -281,7 +281,7 @@ public void testIsNullAndIsNotNull() throws Exception {
   }
 
   @Override protected PCollection<Row> getTestPCollection() {
-    RowType type = RowSqlType.builder()
+    Schema type = RowSqlType.builder()
```

Comment:
Now that we have all type info in core, we should have the builder there as well: `Schema.builder().withInt32().build()`

Reply:
ack